web/cobrands/sass/_dashboard.scss

```diff
@@ -95,6 +95,8 @@
         position: absolute;
         margin-top: -1em;
         margin-#{$right}: 0;
+        right: 0;
+        width: 18%;
```

Comment:
Could we maybe get a comment here to explain the `18%` magic number? I assume it’s the `padding-right: 20%` from `.labelled-line-chart` minus an extra `2%` gutter?